cloud/src/autogluon/cloud/utils/iam.py

```diff
@@ -49,12 +49,11 @@
                 "sagemaker:InvokeEndpoint"
             ],
             "Resource": [
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:artifact/*",
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:transform-job/*",
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:endpoint/*",
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:training-job/*",
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:model/*",
-                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:endpoint-config/*"
+                f"arn:aws:sagemaker:*:{POLICY_ACCOUNT_PLACE_HOLDER}:endpoint/ag-cloudpredictor-serving-trained-model*",
```

Comment:
Seems the prefix is not consistent with the resource created. Shall we just prefix everything with `ag-cloudpredictor*` given the resource name is already in the ARN